module/VuFind/src/VuFind/Db/Service/AccessTokenService.php

```diff
@@ -121,6 +149,18 @@ public function getNonce(int $userId): ?string
      */
     public function deleteExpired(DateTime $dateLimit, ?int $limit = null): int
     {
-        return $this->accessTokenTable->deleteExpired($dateLimit->format('Y-m-d H:i:s'), $limit);
+        $subQueryBuilder = $this->entityManager->createQueryBuilder();
+        $subQueryBuilder->select('at.id')
+            ->from($this->getEntityClass(AccessTokenEnTityInterface::class), 'at')
```

Comment:
There's a minor case-related typo here and in other instances of the interface name -- note the extra capital T in the middle of EnTity.